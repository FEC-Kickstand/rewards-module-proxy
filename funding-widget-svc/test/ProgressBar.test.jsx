import React from 'react';
import renderer from 'react-test-renderer';
import ProgressBar from '../client/components/ProgressBar';

describe('Progress bar', () => {
  test('Value is rendered correctly', () => {
    const tree = renderer.create(<ProgressBar fill={32} goal={64} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
