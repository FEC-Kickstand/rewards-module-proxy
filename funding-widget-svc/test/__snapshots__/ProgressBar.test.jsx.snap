// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress bar Value is rendered correctly 1`] = `
<progress
  className="progressBar"
  max="1"
  value="0.5"
/>
`;
